### ISSUE TITLE
AWS Release v19.0.0

### ISSUE THREAD

**Issue by T-Kukawka** [giantswarm/roadmap#1363]
ℹ️ Team Phoenix is drafting AWS v19.0.0 with Kubernetes v1.24 support.
Target release date: 2023-03-31.
Release PR: ...

## Scope
- [x] https://github.com/giantswarm/roadmap/issues/1362
- [x] https://github.com/giantswarm/roadmap/issues/1102
- [x] Ensure monitoring of k8s components
- [x] upgrade of all components
- [x] [node-local-dns-cache for Cilium#1582](https://github.com/giantswarm/roadmap/issues/1582) - add to release notes
- [x] https://github.com/giantswarm/roadmap/issues/2072
- [x] https://github.com/giantswarm/roadmap/issues/2130
- [x] https://github.com/giantswarm/roadmap/issues/2131
- [x] Fix scraping of k8s node local DNS cache app https://github.com/giantswarm/roadmap/issues/2145#issuecomment-1468143074
- [x] fix cncf test failing due to irsa https://github.com/giantswarm/roadmap/issues/2145#issuecomment-1468111820
- [x] VPA 0.13.0
- [ ] https://github.com/giantswarm/giantswarm/issues/23014
- [x] https://github.com/giantswarm/roadmap/issues/2180
- [x] https://github.com/giantswarm/giantswarm/issues/26243
- [x] https://github.com/giantswarm/giantswarm/issues/26643
- [x] https://github.com/giantswarm/giantswarm/issues/26644
- [x] https://github.com/giantswarm/giantswarm/issues/26645
- [x] https://github.com/giantswarm/roadmap/issues/2407

## Testing Tasks
- [x] https://github.com/giantswarm/giantswarm/issues/25501
- [x] https://github.com/giantswarm/giantswarm/issues/26139
- [x] https://github.com/giantswarm/giantswarm/issues/26149
- [x] https://github.com/giantswarm/roadmap/issues/2128
- [x] https://github.com/giantswarm/roadmap/issues/2354

## Pre-Release
- [ ] Create release ticket on [Releases board](https://github.com/orgs/giantswarm/projects/136)
- [ ] Make an announcement on #sig-product that team is doing a release. Include a link to the ticket from the release board.
- [ ] Make sure release was tested by creating new cluster, upgrading from older version and with HelloWorld app.
- [ ] Aggregate release notes from all changed components and add to `README.md` in the release directory.
- [ ] Send draft release announcement to #sig-product for review ASAP.
- [ ] Verify documentation is up to date.

## Releasing
- [x] Create release PR
[Instructions](https://intranet.giantswarm.io/docs/dev-and-releng/releases/)
    - [x] Define new release in `release.yaml` file.
    - [ ] Activate new release by setting `state` to `active`.
    - [x] Deactivate old release if it's within same major version (`state: deprecated`). Verify with PO if you have doubts.
    - [x] Check versions of all components included. They shouldn't be lower than those in the previous release unless there was a rollback.

[Example PR](https://github.com/giantswarm/releases/pull/318)
- [ ] Release relevant operators.
    - [ ] Tag release branch.
    - [ ] Bump operator version in draft release.
- [x] Make sure that WIP releases in releases repo branch are removed and not merged to master since the master branch is deployed automatically after merging PR and that would make WIP releases visible in customers’ installations which should generally not happen.

## Post-Release
- [ ] Update front-end cluster with the new release.
- [ ] Verify release is available on all installations. You can use [this script](https://github.com/giantswarm/azure-operator/blob/25bf788946c5405532b3b4b76c54b8e685ee291b/scripts/list-releases.sh).
- [ ] Send out release announcement to all customers according to [these instructions](https://intranet.giantswarm.io/docs/dev-and-releng/releases/#release-notes).
- [ ] PO subscribes to release posts on Slack to get notifications if there are follow-up messages.
- [ ] Close release task on Releases board.

**Comment by whites11** [comment]
> E2E tests are blocked by [giantswarm/giantswarm#24407](https://github.com/giantswarm/giantswarm/issues/24407)

not blocked any more thanks to irsa-operator 0.8.5

**Comment by whites11** [comment]
Current blockers:

- https://github.com/giantswarm/prometheus-meta-operator/pull/1049
- https://github.com/cilium/cilium/pull/21366

Nice to have:

- https://github.com/giantswarm/vertical-pod-autoscaler-app/pull/132

**Comment by T-Kukawka** [comment]
The required changes for Cilium should be merged soon:https://github.com/cilium/cilium/pull/21366

However that PR will not fix CNCF tests and this is still needed - https://github.com/cilium/cilium/issues/9207

**Comment by T-Kukawka** [comment]
That's delightful news @aspsk 🎉

**Comment by whites11** [comment]
PR was merged, new 1.11 and 1.12 cilium releases are out but they don't seem to carry the fix

**Comment by T-Kukawka** [comment]
Add to release notes: "Upgrade will take bit longer due to slow and careful nodes rollout for cilium switch"

**Comment by whites11** [comment]
ingress, networkpolicy and cptotcconnectivity tests keep failing. I guess we need to investigate

**Comment by whites11** [comment]
Cncf test green

**Comment by whites11** [comment]
fixed the networkpolicy and cptotcconnectivity tests, ingress seems a flake. running again

**Comment by whites11** [comment]
e2e tests green

**Comment by whites11** [comment]
regarding the temporary release with cilium network policies, last status update from me:

- all e2e tests pass, but the `prometheus` one. turned out I forgot some rules to make scraping work. I now fixed that and e2e tests should be green
- cncf tests are [currently running](https://tekton.giantswarm.io/#/namespaces/test-workloads/pipelineruns/0f4f839c-c8c9-11ed-b214-7a8040c22112?pipelineTask=run-tests&step=run-tests), all good so far

Please note that you can't run tests using tekton for this release, because `standup` (the tool that we use in the tests) does not support any catalog but `default` for the apps in a WC, and this is still not the case for this wip release.
